app/models/internal_note.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class InternalNote < ApplicationRecord
+  belongs_to :document
+  belongs_to :user, optional: true
+  belongs_to :timeline_entry, class_name: "TimelineEntry", foreign_key: :timeline_entries_id, inverse_of: :internal_note
+
+  validates :body, presence: true
```

Comment:
I think we now do all of validation through Requirements in services directory so I think this would be inconsistent here - be fine to just drop it.

Reply:
Yep good point